modules/ROOT/pages/_partials/install-mule-runtime-versions.adoc

```diff
@@ -7,9 +7,19 @@ If you choose to use a different Mule runtime version than the one bundled in yo
 
 [NOTE]
 Studio doesn’t support downgrading Mule runtime to an earlier patch version.
-
 // end::intro[]
 
+// tag::java-mule-runtime-compatibility[]
+If you change the Java or Mule runtime versions in your project, consider these facts:
+
+* Mule runtime 4.5 and earlier works with Java 8 and 11.
```

Comment:
Mule runtime 4.5 is already EoL. Wouldn't be better to mention 4.4 here, which is still on support?

Reply:
Great catch! I'll change it! Thanks!